bin/pulsar-daemon

```diff
@@ -75,6 +70,16 @@ shift
 command=$1
 shift
 
+if [[ "$command" = "bookie" ]]
+then
+  PULSAR_LOG_DIR=${BOOKIE_LOG_DIR:-"$PULSAR_HOME/logs"}
```

Comment:
what happens if a user configured PULSAR_LOG_DIR ?
we are changing the behaviour and people who upgrade may have bad surprises

Reply:
In bkenv.sh, the log dir variable is `BOOKIE_LOG_DIR`,  no `PULSAR_LOG_DIR` variable exists in bkenv.sh. If users config `PULSAR_LOG_DIR` variable in bkenv.sh, default `$PULSAR_HOME/logs` will be used. @eolivelli

Reply:
bkenv.sh 
```
# Log4j configuration file
# BOOKIE_LOG_CONF=
```

Reply:
@eolivelli